ui/src/pages/Configuration/ConfigurationPage.tsx

```diff
@@ -1,11 +1,12 @@
 import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
 
 import { _ } from '@splunk/ui-utils/i18n';
-import TabBar from '@splunk/react-ui/TabBar';
+import TabBar, { TabChangeEvent } from '@splunk/react-ui/TabBar';
```

Comment:
Where did you get it?
<img width="680" alt="image" src="https://github.com/user-attachments/assets/eb878518-cc87-40cc-b96a-ef0f45245a70">